zipkin-receiver-kafka/src/main/scala/com/twitter/zipkin/receiver/kafka/KafkaStreamProcessor.scala

```diff
@@ -1,28 +1,29 @@
 package com.twitter.zipkin.receiver.kafka
 
 import com.twitter.logging.Logger
-import kafka.consumer.KafkaStream
-import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
 import com.twitter.util.{Await, Future}
+import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
+import kafka.consumer.KafkaStream
 
-case class KafkaStreamProcessor(
-  stream: KafkaStream[Option[List[ThriftSpan]], Option[List[ThriftSpan]]],
+case class KafkaStreamProcessor[T](
+  stream: KafkaStream[T, Option[List[ThriftSpan]]],
```

Comment:
its almost like its digging down too deep getting the contents of the Seq instead of the Seq.  Its not obvious where in the KafkaStream or ConsumerIterator its doing this